specification/trace/semantic_conventions/cloudevents.md

```diff
@@ -0,0 +1,51 @@
+# CloudEvents
+
+This document defines how to apply semantic conventions when instrumenting code that uses [CloudEvents](https://cloudevents.io/).
+These semantic conventions are general and not tied to any protocol used to send/receive the events.
+
+**Status**: [Experimental](../../document-status.md)
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Span name](#span-name)
+- [Span kind](#span-kind)
+- [Attributes](#attributes)
+
+<!-- tocstop -->
+
+## Span name
+
+The span name SHOULD be set to the `cloudevents.` prefix, followed by the event type in the following format:
+
+```
+cloudevents.<event type name>
+```
+
+The event type name SHOULD only be used for the span name if it is known to be of low cardinality (cf. [general span name guidelines](../api.md#span)).
+
+Examples:
+
+* `cloudevents.com.github.pull_request.opened`
+
+## Span kind
+
+A producer of an event should set the span kind to `PRODUCER` unless it synchronously waits for a response: then it should use `CLIENT`.
```

Comment:
```suggestion
A emitter of an event should set the span kind to `PRODUCER`.
```

I think waiting for a response event is very unusual, and would probably be handled in another span.

Reply:
Yeah that makes sense. I used `sender` and `receiver` throughout the document, as it's the wording used in the CloudEvents spec. https://github.com/cloudevents/spec/blob/master/primer.md